Product/Production/Deploy/pom.xml

```diff
@@ -479,7 +487,50 @@
         </plugins>
       </build>
     </profile>
-
+    <profile>
+	     <id>SBScripts</id>
+        <activation>
+          <activeByDefault>false</activeByDefault>
+        </activation>
+        <build>
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>sql-maven-plugin</artifactId>
+              <version>1.5</version>
+            <dependencies>
+              <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>5.1.10</version>
```

Comment:
Define version in connect pom and use place holder to display similar like this<version>${mysqldriver.version}</version>